iam_policy_part6.tf

```diff
@@ -0,0 +1,154 @@
+resource "aws_iam_policy" "firefly_readonly_policy_part6" {
+  name        = "FireflyReadonlyPolicy_part6"
+  path        = "/"
+  description = "Read only permission for the cloud configuration Part6"
+
+  policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Action" : [
+          "route53-recovery-readiness:List*",
+          "route53:GetChange",
+          "route53:GetDNSSEC",
+          "route53:GetHealthCheck",
+          "route53:GetHostedZone",
+          "route53:GetReusableDelegationSet",
+          "route53:GetTrafficPolicy",
+          "route53:GetTrafficPolicyInstance",
+          "route53:List*",
+          "route53domains:GetDomainDetail",
+          "route53domains:GetOperationDetail",
+          "route53domains:List*",
+          "route53resolver:GetFirewallDomainList",
+          "route53resolver:GetFirewallRuleGroup",
+          "route53resolver:GetFirewallRuleGroupAssociation",
+          "route53resolver:GetResolverEndpoint",
+          "route53resolver:GetResolverQueryLogConfig",
+          "route53resolver:GetResolverQueryLogConfigAssociation",
+          "route53resolver:GetResolverRule",
+          "route53resolver:GetResolverRuleAssociation",
+          "route53resolver:List*",
+          "s3-outposts:List*",
```

Comment:
s3-outposts:ListAccessPoints is sensitive

Reply:
We are not trying to remove sensitive data, we are trying to minimize the permissions needed by flywheel...